test/distributed/fsdp/test_fsdp_mixed_precision.py

```diff
@@ -388,6 +402,100 @@ def test_mixed_precision_embedding_table(self):
             model.module.run_backward(loss)
             optim.step()
 
+    @skip_if_lt_x_gpu(2)
+    @skipIfNoTorchVision
+    def test_mixed_precision_resnet(self):
+        resnet_model = torchvision.models.resnet50().cuda()
+        resnet_model = nn.SyncBatchNorm.convert_sync_batchnorm(
+            copy.deepcopy(resnet_model),
+            process_group=dist.distributed_c10d._get_default_group()
+        )
+        n_bn = len(
```

Comment:
nit: `sum(1 for m in resnet_model.modules() if isinstance(m, _BatchNorm))` to avoid creating new `nn.Module` references.

This is just for fun/knowledge, but `sys.getsizeof(<some nn.Module>)` returns `48` bytes, while `sys.getsizeof(<some int>)` returns `24` bytes.